frame/test_screencopy_bandwidth.py

```diff
@@ -57,3 +59,25 @@ async def test_inactive_app(self, record_property, server, app) -> None:
         async with server as s, tracker, s.program(app):
             await asyncio.sleep(long_wait_time)
         _record_properties(record_property, server, tracker, 2)
+
+    async def test_app_dragged_around(self, record_property, server) -> None:
```

Comment:
You'll want to limit the servers applicable:

https://github.com/MirServer/mir-ci/pull/17#issuecomment-1500430549

Reply:
I'm having trouble making tests set up that way respect the `-k` argument

Reply:
I've attempted a fix, not sure if that will work or if it's the right way to do it